scripts/test_execution_tutorial.sh

```diff
@@ -8,17 +8,8 @@ SCALDING_VERSION=`cat ${BASE_DIR}/version.sbt`
 SCALDING_VERSION=${SCALDING_VERSION#*\"}
 SCALDING_VERSION=${SCALDING_VERSION%\"}
 
-
-# also trap errors, to reenable terminal settings
-trap onExit ERR
```

Comment:
the onExit was crashing on me on github-actions runner. Anyways, I don't think it's needed as `scald.rb` already cleanups after itself